utilities/ttl/ttl_test.cc

```diff
@@ -87,8 +87,11 @@ class TtlTest : public testing::Test {
   }
 
   void CloseTtl() {
-    delete db_ttl_;
-    db_ttl_ = nullptr;
+    if (db_ttl_ != nullptr) {
+      db_ttl_->Close();
```

Comment:
It might be a good idea to cover both cases. Close() and then destroying DB and destroying the DB without calling Close().

Reply:
Checked in the commit to use `Value-Parameterized Tests` in the test file to consider both cases.